tema2/tema-checker-local.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+
+GOOD_BONUS=`echo -ne "Starting audit...\nAudit done.\n"`
+BAD_BONUS=0
+totalTests=21
+startTest=1
+goodTests=0
+badTests=0
+badHomework=0
+inputName="test"
+existingReadme=0
+badCompile=0
+
+function checkForReadme
+{
+	echo -ne "Checking for README: "
+	readme=$( find -iname "README" -size +0c | wc -l )
+	if [[ $readFrom -eq  1 ]]; then
+		echo -ne "- OK! \n"
+		existingReadme=10
+	else
+		readme=$( find -iname "README.md" -size +0c | wc -l )
+		if [[ $readme -eq  1 ]]; then
+			echo -ne "- OK! \n"
+			existingReadme=10
+		else
+			readme=$( find -iname "README.txt" -size +0c | wc -l)
+			if [[ $readme -eq  1 ]]; then
+				echo -ne "- OK! \n"
+				existingReadme=10
+			else
+				echo -ne "- NO README! \n"
+				existingReadme=0
+			fi
+		fi
+	fi
+}
+
+function checkCodeStyle
+{
+	java -jar checker/checkstyle/checkstyle-7.3-all.jar -c checker/checkstyle/poo_checks.xml * > checkstyle.txt
+
+	YOUR_BONUS=`cat checkstyle.txt`
```

Comment:
rename bonus in codestyle, nu mai este dat ca bonus (asa era prima oara cand am introdus checkstyle)